django-tuttle/provider/management/commands/fetch_repositories.py

```diff
@@ -66,15 +66,13 @@ def handle(self, *args, **options):
                             'key': key.key,
                             'repository': Repository.objects.get(name=repo.name),
                         }
-                        # if key is not null, DeployKey object can be created using the previous params
-                        if key:
-                            try:
-                                DeployKey.objects.get_or_create(**params)
+                        try:
+                            DeployKey.objects.create(**params)
 
-                            except DeployKey.MultipleObjectsReturned:
-                                msg = 'More than 1 Deploykey with this params: %s ' % params
-                                logger.error(msg)
-                                raise CommandError(msg, 'Fix the database')
+                        except DeployKey.MultipleObjectsReturned:
```

Comment:
You don't need to catch MultipleObjectsreturned here as you are not using get_or_create anymore. Catch the right exception